### PR TITLE
fix: screenDensity is double

### LINKED ISSUE #53521: [sentry] screenDensity has to be a floating point number
[sentry](https://pub.dev/packages/sentry) v3.0.1

Right now when I try to use `Device` context `screenDensity` parameter, I get the following error on sentry.io:

```
There was 1 error encountered while processing this event
contexts.device.screen_density: Discarded invalid value
```

Reason | expected a floating point number
-- | --
Value | 2.625

### PR DESCRIPTION
**Pull request by bruno-garcia**
Resolves flutter/flutter#53521

### CODE REVIEW

**Comment by bruno-garcia** [comment]
image

Why isn't the status check updating?

File: lib/src/base.dart [hunk 1]
```diff
@@ -659,7 +659,7 @@ class Device {
   final String screenResolution;
 
   /// A floating point denoting the screen density.
-  final String screenDensity;
+  final double screenDensity;
```
Comment (yjbanov) [inline lib/src/base.dart:662]: This is a breaking change. Let's do a major version bump.

Comment (bruno-garcia) [reply lib/src/base.dart:662]: That's true. Should I bump it as part of this PR?

Comment (yjbanov) [reply lib/src/base.dart:662]: Yep.

Comment (bruno-garcia) [reply lib/src/base.dart:662]: Bumped the major and added changelog.

**Review by yjbanov — APPROVED** [review]

**Pull request merged**